.github/workflows/build.yml

```diff
@@ -1,35 +1,34 @@
-# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
-
-name: Build
+name: Lint Publish
```

Comment:
I would prefer to match the workflow name (currently: `Lint Publish`) with the filename (currently: `build.yml`) .

There are two options:
1. rename the file `.github/workflows/build.yml` to `.github/workflows/lint-publish.yml`
2. change the value of `name` in the Workflowto `Build`

Reply:
After talking to Steffen, it turns out that the renaming was done accidentally. There is no linitng done in the Workflow yet, therefore the workflow should be called `Publish`.
I applied following changes
* Renamed the file to `.github/workflows/publish.yaml`
* Renamed the workflow to `Publish`